src/components/ErrorBoundary/ErrorBoundary.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import * as logging from '../../Logging';
+import ErrorBoundary from './index';
+
+global.newrelic = {
+  addPageAction: jest.fn(),
+  noticeError: jest.fn(),
+};
+
+const welcomeError = new Error('Catch me if you can!');
```

Comment:
This variable is only used in 1 place right now. Should we move its declaration into the test that uses it?